bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -178,10 +173,28 @@ class ApiVersionedDir {
   // Names of *_gapic_assembly_* rules (since they may be overridden by the user)
   private final Map<String, String> assemblyPkgRulesNames = new HashMap<>();
 
-  // Attributes of *_gapic_library rules to be overridden
+  // Attributes of *_gapic_library rules to be overridden. The keys for these maps are the names of
+  // the library rules.
   private final Map<String, Map<String, String>> overriddenStringAttributes = new HashMap<>();
   private final Map<String, Map<String, List<String>>> overriddenListAttributes = new HashMap<>();
 
+  ApiVersionedDir() {
+    // Multiple languages:
+    PRESERVED_PROTO_LIBRARY_STRING_ATTRIBUTES.put("package_name", null);
+    PRESERVED_PROTO_LIBRARY_STRING_ATTRIBUTES.put("transport", null);
+    PRESERVED_PROTO_LIBRARY_STRING_ATTRIBUTES.put("rest_numeric_enums", "False");
```

Comment:
would it be a string `"False"`, or a boolean `False`? I feel there is a possibility of an inconsistency here.

Reply:
Or, if I'm guessing correctly, for a string, it should be `"\"False\""`?

Reply:
After looking at the remaining code, I must say it is still unclear to me. Would it be possible to inspect the actual content of the build file that had `False` value added? Something tells me that it would be a string `"False"` rather than a boolean.

Reply:
This is intended to be a boolean value as per the internal design doc, which is reflected in the [Go](https://github.com/googleapis/gapic-generator-go/blob/4f333a3e023bdc6e384a8bc95409ba48f7b04727/rules_go_gapic/go_gapic.bzl#L96) and [Python](https://github.com/googleapis/gapic-generator-python/blob/8839c6fc593d62a10a069cb28f72cd76f080ecc5/rules_python_gapic/py_gapic.bzl#L46) implementations. For sample files, you can look at the mustache and baseline files in this PR. Or were you looking for something else?

Reply:
Definitely a boolean `False`.